integration/async-iterable-services/simple.ts

```diff
@@ -101,7 +101,9 @@ export interface Echoer {
 
 export class EchoerClientImpl implements Echoer {
   private readonly rpc: Rpc;
-  constructor(rpc: Rpc) {
+  private readonly service: string;
+  constructor(rpc: Rpc, service?: string) {
```

Comment:
@paralin wdyt about a `constructor(rpc: Rpc, opts?: { service?: string })` so that we could add more opts in the future w/o a breaking api change?

I'm pretty happy to ship as-is and then just make the opts change later if ever needed.

Reply:
I'm happy with either, opts works for me.

Reply:
@stephenh I updated the PR to match your suggestion! 🚀